test/visual/svg_import.js

```diff
@@ -2,7 +2,6 @@
   fabric.config.configure({
     enableGLFiltering: false
   });
-  fabric.Object.ownDefaults.objectCaching = true;
```

Comment:
yeah, this proves that qunit is full of sideeffects between tests
that I had to remove this
because some previous test in another suite had this flipped

Reply:
I grew my js expertise and test understanding working of fabricjs during the years before professional js was a carreerr for me. I m sure code written between 2015-2018 is full of poor decisions based on 'i need this to work for work related issues' and 'tests is better than no tests'